model/event/event_finder.go

```diff
@@ -161,9 +161,9 @@ func TaskEventsInOrder(id string) db.Q {
 // Distro Events
 
 // FindLatestPrimaryDistroEvents return the most recent non-AMI events for the distro.
-func FindLatestPrimaryDistroEvents(id string, n int) ([]EventLogEntry, error) {
+func FindLatestPrimaryDistroEvents(id string, n int, before *time.Time) ([]EventLogEntry, error) {
```

Comment:
I'm assuming before is used for pagination? Can we add that to the function comment?